src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/HeaderSWA.tsx

```diff
@@ -54,7 +57,11 @@ export const HeaderSWA = () => {
                 as={RouterLink}
                 width="100%"
                 variant="primaryBlack"
-                to="/sell/submission"
+                to={
+                  enableNewSubmissionFlow
+                    ? "sell2/submissions/new"
```

Comment:
are we planning to release the feature with the rout `sell2` and rename to `sell` after we remove the feature flag logic?

Reply:
good point - I think we can update the route when we plan on releasing instead of using a ff here

Reply:
The reason why I added the feature flag is to get the flow ready for the QA sessions. Like this, we can test the complete flow, from clicking the "Start Selling" button until the end without any confusion.